engine/gameobject/src/gameobject/gameobject.cpp

```diff
@@ -2491,7 +2492,69 @@ namespace dmGameObject
             }
 
             if (!DispatchMessages(collection, &collection->m_ComponentSocket, 1))
+            {
                 ret = false;
+            }
+        }
+
+        if (update_context->m_FixedUpdateFrequency != 0)
+        {
+            const float time = collection->m_FixedAccumTime + update_context->m_DT; // Add the scaled time
+            const float fixed_frequency = update_context->m_FixedUpdateFrequency;
+            // If the proxy is slowed down, we want e.g. the physics to be slowed down as well
+            const float fixed_dt = (1.0f / (float)fixed_frequency) * update_context->m_TimeScale;
```

Comment:
We scale the fixed_dt, so that it follows the same time scale as the rest of the collection.